src/test/kotlin/no/nav/familie/ef/sak/config/PdlClientConfig.kt

```diff
@@ -8,22 +8,106 @@ import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.context.annotation.Primary
 import org.springframework.context.annotation.Profile
+import java.time.LocalDate
+import java.time.LocalDateTime
 
 @Configuration
 @Profile("mock-pdl")
 class PdlClientConfig {
 
+    private val startdato = LocalDate.of(2020, 1, 1)
+    private val sluttdato = LocalDate.of(2021, 1, 1)
+
     @Bean
     @Primary
     fun pdlClient(): PdlClient {
         val pdlCLient: PdlClient = mockk()
+
+        val folkeregisterpersonstatus = listOf(Folkeregisterpersonstatus("bosatt", "bosattEtterFolkeregisterloven"))
+        val navn = listOf(Navn("Fornavn", "mellomnavn", "Etternavn"))
+        val kjønn = listOf(Kjønn(KjønnType.KVINNE))
+        val adressebeskyttelse = listOf(Adressebeskyttelse(gradering = AdressebeskyttelseGradering.UGRADERT))
+
         every { pdlCLient.hentSøkerKort(any()) } returns
-                PdlSøkerKort(adressebeskyttelse = listOf(Adressebeskyttelse(gradering = AdressebeskyttelseGradering.UGRADERT)),
+                PdlSøkerKort(adressebeskyttelse = adressebeskyttelse,
                              dødsfall = emptyList(),
-                             folkeregisterpersonstatus = listOf(Folkeregisterpersonstatus("bosatt",
-                                                                                          "bosattEtterFolkeregisterloven")),
-                             kjønn = listOf(Kjønn(KjønnType.KVINNE)),
-                             navn = listOf(Navn("Fornavn", "mellomnavn", "Etternavn")))
+                             folkeregisterpersonstatus = folkeregisterpersonstatus,
+                             kjønn = kjønn,
+                             navn = navn)
+
+        every { pdlCLient.hentSøker(any()) } returns
+                PdlSøker(
+                        adressebeskyttelse = adressebeskyttelse,
+                        bostedsadresse = bostedsadresse(),
+                        dødsfall = listOf(),
+                        familierelasjoner = listOf(),
+                        fødsel = listOf(),
+                        folkeregisterpersonstatus = folkeregisterpersonstatus,
+                        fullmakt = fullmakter(),
+                        kjønn = kjønn,
+                        kontaktadresse = kontaktadresse(),
+                        navn = navn,
+                        opphold = listOf(),
+                        oppholdsadresse = listOf(),
+                        sivilstand = sivilstand(),
+                        statsborgerskap = statsborgerskap(),
+                        telefonnummer = listOf(Telefonnummer(landskode = "+47", nummer = "98999923", prioritet = 1)),
```

Comment:
landskode -> landkode

Reply:
Dette er en del av Pdl domeneobjekt, og den bruker landskode. Jag kan bruke @JsonProperty for å overridea den